Xerces library link broken

Hello guys, I was encountered with problem during compilation.
Here what `./configure` said to me:
```
checking for Xerces C++ Parser headers in /usr/src/cc/CodeCompass-deps/include and /usr/src/cc/CodeCompass-deps/include/xercesc... not found
checking for Xerces C++ Parser... no
configure: error: cannot find the Apache Xerces library
```
I strictly follow instructions from this page: https://github.com/Ericsson/CodeCompass/wiki/Lazy-Dog.
And dependencies building finished without problems.
But it looks like there is really no xerces includes in CodeCompass-deps tree.

UPD:
I examined `build-deps.sh` and found that link in `build_xercesc` is broken http://xenia.sote.hu/ftp/mirrors/www.apache.org/xerces/c/3/sources/xerces-c-3.1.1.tar.gz, returns 404 result.

--- a/scripts/build_deps.sh
+++ b/scripts/build_deps.sh
@@ -859,7 +859,7 @@ build_xercesc()
 {
   dep gcc
   
-  download http://xenia.sote.hu/ftp/mirrors/www.apache.org//xerces/c/3/sources/xerces-c-3.1.1.tar.gz 
+  download http://archive.apache.org/dist/xerces/c/3/sources/xerces-c-3.1.1.tar.gz
   compile xercesc xerces-c-3.1.1 bin/SAX2Print
 }
 

